src/wallet.c

```diff
@@ -235,24 +235,41 @@ int wallet_generate_node(const char *passphrase, const char *entropy, HDNode *no
 }
 
 
-void wallet_report_xpub(const char *keypath, char *xpub)
+int wallet_report_public(const char *keypath, char *xpub, char *pub65, char *chaincode)
```

Comment:
I will revert this change. I added it because I did not want to add dependencies in MEW. However, I found that they do already have a module for de-serializing an xpub and that a compressed and its corresponding uncompressed pubkey gives the same resulting address.